### PR TITLE
Add spritesheet and tilesheet, draw background, monogame extended and refreshed README.md

### PR DESCRIPTION
**Pull request by superboum**
- Improve README
  - Add Monogame Extended as dependency which add support for (but not yet implemented):
    -  Camera2d
    - TextureAtlases
    - Viewport Adapters
    - etc.
  - Add tilesheet and spritesheet from the PiratePack
  - Draw a map with only ocean

### CODE REVIEW

File: TarponSimulator2017/Core/Physics.cs [hunk 1]
```diff
@@ -59,20 +59,16 @@ public Physics ()
...
 			Orientation = new Vector2 (0, -1);
-
-
-
 		}
 
 		public float OrientiationFloat ()
 		{
-			return (float)Math.Atan2 (Orientation.X, -Orientation.Y);
+			return (float) (Math.Atan2 (Orientation.X, -Orientation.Y) + Math.PI);
```
Comment (Hamilcare) [inline TarponSimulator2017/Core/Physics.cs]: Why add Math.PI ?

Comment (superboum) [reply TarponSimulator2017/Core/Physics.cs]: Nice shot!
The explanation is a bit long but interesting for furture development.
I use the output of `OrientiationFloat` to compute the rotation of the sprite (the png representing the boat).
So when `OrientiationFloat` output a value of `0f`, the sprite is not rotated.
But `Math.Atan2 (Orientation.X, -Orientation.Y)` equals 0 when `x = 0, y = -1`.
So, our boat will move to the top when we will accelerate with a rotation of 0.
The previous sprite was a boat pointing to the top, so no problem here.
But the [new spritesheet](https://github.com/Hamilcare/TarponSimulator2017/blob/cbefbac4b4f71b953b5e85c6c9f696b2fb0de9e8/TarponSimulator2017/Content/img/spritesheet.png) has ships stored pointing to the bottom of the screen.
So, to draw a ship that points to the top, I must rotate it by `90 deg` or `Math.PI rad`.

After that small explanation, I am realizing that this shift should be put in the Draw part, as it is sprite dependant. I will fix that as soon as possible.

Comment (Hamilcare) [reply TarponSimulator2017/Core/Physics.cs]: Thanks for the explanation, that makes sense :)

File: README.md [hunk 2]
```diff
@@ -2,59 +2,56 @@
 
 [![Build Status](https://ci.deuxfleurs.fr/job/TarponSimulator2017/job/master/badge/icon)](https://ci.deuxfleurs.fr/job/TarponSimulator2017/job/master/)
 
-## Download
+## Play it!
 
-[Download Tarpon Simulator 2017 binaries (Windows and Linux)](https://ci.deuxfleurs.fr/job/TarponSimulator2017/job/master/lastSuccessfulBuild/artifact/TarponSimulator2017.zip)
+ 1. Install dependencies
+   * On Linux, you must install SDL2 and mono. On Fedora: `dnf install mono SDL2 SDL2-devel`
+   * On macOS, you must install SDL2 and mono, we don't know how.
```
Comment (Hamilcare) [inline README.md]: I Guess we could say that tarpon  compatibility with macOS is equivalent to openFrameworks compatibility with Linux, so tarpon is definitely supported on macOS :)
It's the right place to troll isn't it ?

File: TarponSimulator2017/Draw/MapDrawer.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+			Texture = content.Load<Texture2D> ("Content/img/tilesheet");
+			TextureRegion = new Rectangle (1024, 512, 128, 128);
+		}
+
+		public void Draw(SpriteBatch spriteBatch, GameTime gameTime)
+		{
+			for (int i = 0; i < width; i += 128) {
+				for (int j = 0; j < height; j += 128) {
+					spriteBatch.Draw (
+						Texture,						// Texture of the boat
```
Comment (Hamilcare) [inline TarponSimulator2017/Draw/MapDrawer.cs]: Beware of hidden comments during copy paste ;)
If i get it, you draw the background using a square of texture of 128 pixels is that right

**Pull request merged**